src/backend/distributed/deparser/deparse_table_stmts.c

```diff
@@ -229,6 +229,16 @@ AppendAlterTableCmdAddConstraint(StringInfo buf, Constraint *constraint)
 
 		appendStringInfoString(buf, " )");
 	}
+
+	if (constraint->deferrable)
+	{
+		appendStringInfoString(buf, " DEFERRABLE");
+	}
+
+	if (constraint->initdeferred)
```

Comment:
You can put this if statement inside the previous one

Reply:
Good idea.